src/table/table-role/__integ__/grid-navigation.test.ts

```diff
@@ -28,9 +34,56 @@ test(
   })
 );
 
+test(
+  'cell focus stays in the same position when row gets removed with auto-refresh',
+  useBrowser({}, async browser => {
+    const page = new BasePageObject(browser);
+    await browser.url('#/light/table-fragments/grid-navigation-custom');
+    await page.waitForVisible('table');
```

Comment:
Minor: Don't we normally abstract this into something like a `setupTest` function to remove some of this duplication? Although since it just a test file I don't have any problem with the duplication 🤷🏻‍♂️

Reply:
Makes sense, will update!